internal/lookout/ui/src/components/job-dialog/JobDetails.tsx

```diff
@@ -82,19 +82,20 @@ export default function JobDetails(props: DetailsProps) {
           onToggleExpand={toggleExpanded}
         />
       )}
-      <ContainerDetails jobYaml={props.job.jobYaml} />
-
       {props.job.jobYaml && (
-        <div className="details-yaml-container">
-          <Accordion>
-            <AccordionSummary expandIcon={<ExpandMore />}>
-              <h3>Job YAML</h3>
-            </AccordionSummary>
-            <AccordionDetails>
-              <p className="details-yaml">{props.job.jobYaml}</p>
-            </AccordionDetails>
-          </Accordion>
-        </div>
+        <>
```

Comment:
What does adding an empty, unnamed tag here accomplish?

Reply:
https://reactjs.org/docs/fragments.html#short-syntax
An expression must have a single React element. In this case we need to return two, so am wrapping them in a Fragment